### PR TITLE
Feature/mob 3405

### PR DESCRIPTION
**Pull request by ancafeurdean09**
## JIRA
[MOB-3405](https://karhoo.atlassian.net/browse/MOB-3405)
[MOB-3393](https://karhoo.atlassian.net/browse/MOB-3393)

## DESCRIPTION
[Create a description of what you have changed in this branch]

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [ ] New feature 
- [ ] Tech Debt
- [x] Breaking change 
- [ ] Documentation Update
- [x] Other: *UI redesign*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [ ] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteCell/QuoteViewModel.swift [hunk 1]
```diff
@@ -9,34 +9,125 @@
...
 import KarhooSDK
 
+enum VehicleTag: String {
+    case electric, hybrid, wheelchair, childSeat, taxi, executive
+    
+    enum CodingKeys: String, CodingKey {
+        case childSeat = "child-seat"
+    }
+    
+    var title: String {
```
Comment (dianakarh) [inline KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteCell/QuoteViewModel.swift:19]: Shouldn't this be localized?

File: KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteCell/QuoteViewModel.swift [hunk 2]
```diff
@@ -9,34 +9,125 @@
...
+    
+    enum CodingKeys: String, CodingKey {
+        case gpsTracking = "gps_tracking"
+        case flightTracking = "flight_tracking"
+        case trainTracking = "train_tracking"
+//        case driverDetails = "driver_details"
+//        case vehicleDetails = "vehicle_details"
+    }
+    
+    var title: String {
```
Comment (dianakarh) [inline KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteCell/QuoteViewModel.swift:66]: Same question as for the VehicleType enum regarding localization

Comment (mohadian) [reply KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteCell/QuoteViewModel.swift:66]: Here is the resources for Android:
https://github.com/karhoo/karhoo-android-ui-sdk/blob/18e335e031ee036f62bd36f80efdbdd88217a9ab/uisdk/src/main/res/values/strings.xml#L22

File: KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteCell/QuoteViewModel.swift (hunk 1 above)
Comment (ancafeurdean09) [reply KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteCell/QuoteViewModel.swift:19]: added localizations here now

File: KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteCell/QuoteViewModel.swift (hunk 2 above)
Comment (ancafeurdean09) [reply KarhooUISDK/Screens/BookingScreen/QuoteListMVP/QuoteCell/QuoteViewModel.swift:50]: added localised strings here now

**Review by dianakarh — DISMISSED** [review]

**Review by dianakarh — APPROVED** [review]

**Pull request merged**